src/main/resources/static/assets/js/sdh.js

```diff
@@ -44,7 +44,10 @@ class ModalEditRow {
         	<i class="fas fa-angle-right"></i>
         	</b-form-checkbox>
       	  </template>
-      	  <template slot="edit" slot-scope="row">
+       	  <template slot="flagged" slot-scope="row">
+       	  	<span :class="{'flag-enabled':(row.item.flagged), 'flag-disabled':(!row.item.flagged)}" @click="toggleFlag(row)"><i class="fas fa-exclamation-circle"></i></span>
```

Comment:
There are a couple of other ways of doing this that might be cleaner. First, you could use the array syntax for style bindings:

```html
<span :class="[row.item.flagged ? 'flag-enabled' : 'flag-disabled']">...</span>
```

Another idea would be to use the pattern that Bootstrap uses for buttons, where you have a `flag` class and a `flag-enabled` class.

```css
.flag {
  color: #ced4da;
}
.flag.flag-enabled {
  color: #bd3737;
}
```

```html
<span class="flag" :class="{'flag-enabled': (row.item.flagged)}">...</span>
```

Reply:
Oh thanks. Those are both much nicer. I'll use the second approach since it aligns more with Bootstrap conventions.